File: skeleton/src/main/java/com/example/skeleton/Skeleton.java
package com.example.skeleton;

import android.view.View;

import androidx.recyclerview.widget.RecyclerView;

import com.example.skeleton.widget.RecyclerViewSkeletonScreen;
import com.example.skeleton.widget.ViewSkeletonScreen;

/**
 * Created by Charles
 */

public class Skeleton {

    public static RecyclerViewSkeletonScreen.Builder bind(RecyclerView recyclerView) {
        return new RecyclerViewSkeletonScreen.Builder(recyclerView);
    }

    public static ViewSkeletonScreen.Builder bind(View view) {
        return new ViewSkeletonScreen.Builder(view);
    }

}

File: duskeleton/src/main/java/com/example/duskeleton/Skeleton.kt
package com.example.duskeleton

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import com.example.duskeleton.widget.ViewSkeletonScreen
import com.example.duskeleton.widget.recycler.RecyclerViewSkeletonScreen

/**
 * Author: Charles
 * Version V1.0
 * Date: 2020/8/10
 * Description:
 * Modification History:
 * Date Author Version Description
 */
object Skeleton {
    fun bind(recyclerView: RecyclerView): RecyclerViewSkeletonScreen.Builder {
        return RecyclerViewSkeletonScreen.Builder(recyclerView)
    }

    fun bind(view: View): ViewSkeletonScreen.Builder {
        return ViewSkeletonScreen.Builder(view)
    }

}
